provisioning/roles/install-project-dependencies/tasks/main.yml

```diff
@@ -1,2 +1,5 @@
 - name: install node modules
-  npm: path=/vagrant
\ No newline at end of file
+  npm: path=/vagrant
+
+- name: install python unittest2
+  pip: name=unittest2
```

Comment:
Maybe we could use `requirements.txt`? The [pip module](https://docs.ansible.com/ansible/pip_module.html) supports it as a parameter:

```
- pip: requirements=/my_app/requirements.txt
```